server/src/MessageHandler.ts

```diff
@@ -0,0 +1,149 @@
+import {
+  TextDocumentPositionParams,
+  CancellationToken,
+  TextDocuments,
+  DocumentFormattingParams,
+  TextEdit,
+  Range,
+  Location,
+  DeclarationParams,
+  CompletionParams,
+  CompletionItem,
+  CompletionItemKind,
+} from 'vscode-languageserver'
+import * as util from './util'
+import { fullDocumentRange } from './provider'
+import { getDMMF } from '@prisma/sdk'
+import { TextDocument, Position } from 'vscode-languageserver-textdocument'
+import format from './format'
+
+export class MessageHandler {
+  constructor() {}
+
+  async handleHoverRequest(
+    params: TextDocumentPositionParams,
+    documents: TextDocuments<TextDocument>,
+  ) {
+    return null
+  }
+
+  getWordAtPosition(document: TextDocument, position: Position): String {
+    let currentLine = document.getText({
+      start: { line: position.line, character: 0 },
+      end: { line: position.line, character: 9999 },
+    })
+    // search for the word's beginning and end
+    var beginning = currentLine.slice(0, position.character + 1).search(/\S+$/)
+    var end = currentLine.slice(position.character).search(/\W/)
+    if (end < 0) {
+      return ''
+    }
+    return currentLine.slice(beginning, end + position.character)
+  }
+
+  async handleDefinitionRequest(
+    documents: TextDocuments<TextDocument>,
+    params: DeclarationParams,
+    _token?: CancellationToken,
+  ): Promise<Location> {
+    // bad workaround!
+    // ASTNode will be necessary to handle this a lot better
```

Comment:
```suggestion
    // TODO: Replace bad workaround as soon as ASTNode is available
```

(no idea if I used `ASTNode` correctly here - gist: a TODO will ensure we look at it again in the future, or at least make more probably)